package.json

```diff
@@ -128,14 +128,15 @@
     "sinon-chai": "3.5.0",
     "solhint": "3.0.0",
     "truffle": "5.1.53",
-    "ts-node": "8.6.2",
     "ts-loader": "^9.2.3",
+    "ts-node": "8.6.2",
     "typechain": "1.0.5",
     "typechain-target-trufflehotfix": "0.0.4-alpha",
-    "typescript": "3.8.3",
+    "typescript": "^4.3.5",
     "web3-provider-engine": "16.0.1",
-    "webpack": "4.37.0",
-    "webpack-cli": "3.3.6"
+    "webpack": "^5.43.0",
+    "webpack-cli": "^4.7.2",
+    "webpack-node-externals": "^3.0.0"
```

Comment:
any reason for these?

Reply:
Basically we had an old version of webpack and typescript that was causing troubles with the new modules when we integrate to the main repository. Also having old version of webpack and typescript libraries could be problematic (typing errors, outdated rules, missing ecmascript support, etc)

Reply:
gotcha